### DISCUSSION TITLE
NASA Posters - Adapted for eink

### DISCUSSION THREAD

**Discussion by mergen3107** [koreader/koreader#9380] [Show and tell]
Hi guys!
I have eink-adapted NASA Poster of Visions of Future from here: https://www.jpl.nasa.gov/galleries/visions-of-the-future

I upscaled them with http://waifu2x.udp.jp/ and then added dithering with https://imagemagick.org/ . For dithering help, thanks to @NiLuJe .

Here is the link to archives with two resolutions, 1404x1872 (most 7.8 and 10.3 inch readers), and 1072x1448 and 1440x1080 (most 6 inch readers): https://disk.yandex.com/d/4MPiBsK7SIyIOA
They also contains the script I used so you can do the same for other resolutions if needed. Posters are stretched to fit the whole screen area. If you want to preserve the aspect ratio, remove "!" after the two resolutions in the script.

They look gorgeous of the eink screen! I hope someone will find those nice!
Cheers!

![photo_2022-07-28_09-58-10](https://user-images.githubusercontent.com/6279855/181418413-309a516f-80b3-4d91-8b6f-d5b7f592cac1.jpg)

**Comment by Kheperure** [comment]
These are gorgeous and fun.  Thanks!

**Reply by mergen3107** [reply]
Enjoy :D

**Comment by SteadyQuad** [comment]
Just found this, nicely done! 

FYI There's high resolution (6000x9000) TIFF versions available too (click a poster to open the poster in the gallery, at the bottom of the page is a 'full image details' link, from which you can choose to download a TIFF version, see e.g. [here](https://www.jpl.nasa.gov/images/the-grand-tour-jpl-travel-poster ). It may not make a ton of difference in the grayscale dithered end result, but since you went through the trouble of upscaling the JPG's first, this might be even better. (The TIFF's are huge files btw).

**Reply by mergen3107** [reply]
Thanks! Oh, great, I could have used those instead :D
